plugins/doc_fragments/eos.py

```diff
@@ -11,6 +11,14 @@ class ModuleDocFragment(object):
 
     # Standard files documentation fragment
     DOCUMENTATION = r"""options:
+  version:
+    description:
+    - Specifies the version for network_cli or httpapi connections.
```

Comment:
```suggestion
    - Specifies the version of the JSON response returned when I(output=json).
```

Reply:
@Qalthos - I did not understand the meaning of end of the helpstring 'when I'?

Reply:
I have made the necessary changes.